Scripts/DCS-BIOS/test/compile/LocalCompile.lua

```diff
@@ -0,0 +1,27 @@
+-- This can be used to run the DCS-BIOS JSON generation locally without
+-- starting a DCS mission
+-- DCS-BIOS wiki have information on how to install the lua interpreter. No need to install
+-- additional lua modules such as socket etc.
+-- Execute => lua LocalCompile-lua
+
+local dcs_variant = "DCS"
+DCS_USER_DIR_ROOT = os.getenv("USERPROFILE") .. "\\Saved Games\\" .. dcs_variant .. "\\"
```

Comment:
As above, I think we want to decouple this from a machine with a dcs install so we can run it in github actions

Reply:
OK

Reply:
How do you know the paths for the action on Github. Is root the root of the repo?